package-lock.json

```diff
@@ -18428,4 +18428,4 @@
       "integrity": "sha512-V50KMwwzqJV0NpZIZFwfOD5/lyny3WlSzRiXgA0G7VUnRlqttta1L6UQIHzd6EuBY/cHGfwTIck7w1yH6Q5zUw=="
     }
   }
-}
+}
```

Comment:
Please enter a blank line at the end of file, its a convention.

Reply:
I made the changes then made commit and pushed it. Is this okay or I need to create a new pr?